tests/testsuite/build.rs

```diff
@@ -5672,7 +5672,6 @@ fn close_output() {
 hello stderr!
 [ERROR] [..]
 [WARNING] build failed, waiting for other jobs to finish...
-[ERROR] [..]
```

Comment:
The error going away on this line is the "build failed" error from the following line. It's preceded by the actual error, and a "build failed, waiting for other jobs to finish..." message.

https://github.com/rust-lang/cargo/blob/2ac382d8e2d0bcf228ab2fae53efbaf5d2b81361/src/cargo/core/compiler/job_queue.rs#L900